react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -215,10 +221,10 @@ class AddPeopleDialog extends Component<*, *> {
             });
 
             this.props._conference
-                && inviteRooms(
-                    this.props._conference,
-                    this.state.inviteItems.filter(
-                        i => i.type === 'videosipgw'));
+                && this.props.inviteRooms(
```

Comment:
There is a type "room" and the function is called inviteRooms, but you filter for videosipgw. What is the difference between room and videosipgw?

Reply:
The invitation works with 3 types returned - user, room and videosipgw. User is a regular participant, room is like a chat room with multiple participants like in Stride. And also we  videosipgw. So let's say we have chat room (type===room) and conference room (type=== videosipgw).
I agree its a little confusing and I also was wondering about that. If you have an idea how to improve it?

Reply:
Maybe we can imagine what it would be like with different names, like specifically calling them chatRoom and conferenceRoom/videoRoom/sipGW instead of using the general term room.

Reply:
Ok, I will address that.